### PR TITLE
refactor(sample_sensor_kit_launch): add use_gnss_ins_orientation parameter

### PR DESCRIPTION
**Pull request by meliketanrikulu**
Signed-off-by: melike tanrikulu <melike@leodrive.ai>

## Description

This PR edits the launch file based on the latest changes in the gnss_poser package. The use_gnss_ins_orientation parameter has been added instead of the use_ublox _receiver parameter.
## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: sample_sensor_kit_launch/launch/gnss.launch.xml [hunk 1]
```diff
@@ -34,7 +34,7 @@
       <arg name="output_topic_gnss_fixed" value="fixed"/>
 
       <arg name="coordinate_system" value="$(var coordinate_system)"/>
-      <arg name="use_ublox_receiver" value="true"/>
+      <param name="use_gnss_ins_orientation" value="true"/>
```
Comment (YamatoAndo) [inline sample_sensor_kit_launch/launch/gnss.launch.xml]:
```suggestion
      <arg name="use_gnss_ins_orientation" value="true"/>
```

Comment (YamatoAndo) [reply sample_sensor_kit_launch/launch/gnss.launch.xml]: updated https://github.com/autowarefoundation/sample_sensor_kit_launch/pull/44/commits/2c8f990c92096cb81ecae8ed53e3260a03d1eff2

**Review by YamatoAndo — APPROVED** [review]

**Comment by YamatoAndo** [comment]
@meliketanrikulu May I merge this PR?

**Comment by meliketanrikulu** [comment]
> @meliketanrikulu May I merge this PR?

Yes please.. Thank you..

**Pull request merged**